check_each_commit.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
I don't think this file needs to be preserved forever, since it was a one-off, so I'll make sure to remove it too before merging